examples/docs_snippets/docs_snippets_tests/guides_tests/managing_ml_tests/managing_ml_tests.py

```diff
@@ -0,0 +1,32 @@
+# isort: skip_file
+
+from dagster._core.definitions.load_assets_from_modules import assets_from_modules
+from dagster._core.definitions.materialize import materialize
```

Comment:
for public APIs, it's generally better to import from the top-level Dagster package:
```python
from dagster import materialize
```

Reply:
thanks- copied this from somewhere :)